src/views/Layout.jsx

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 import { Outlet } from 'react-router-dom';
 import { useAuth, SignInButton, SignOutButton } from '../api/useAuth';
+import { NavLink } from 'react-router-dom';
```

Comment:
It looks like the import from react-router-dom is duplicated, the first import is on line 2, second import is on line 4, could be worth combining?